src/middlewared/middlewared/api/v26_04_0/disk.py

```diff
@@ -205,3 +217,45 @@ class DiskWipeArgs(BaseModel):
 class DiskWipeResult(BaseModel):
     result: None
     """Returns `null` when the disk wipe operation is successfully started."""
+
+
+@single_argument_args('disk_sed_unlock')
+class DiskUnlockSedArgs(BaseModel):
+    name: NonEmptyString
+    """Name of disk to unlock."""
+    password: Secret[NonEmptyString] = NotRequired
```

Comment:
Why can't it be 
```suggestion
    password: Secret[NonEmptyString | None] = None
```
?